<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="6" hostname="Manjus-MacBook-Pro.local" name="BranchIO.com.HomePageTest" tests="7" failures="0" timestamp="14 Sep 2018 07:45:29 GMT" time="0.009" errors="1">
  <testcase name="setupEn" time="0.009" classname="BranchIO.com.HomePageTest">
    <error type="java.io.FileNotFoundException" message="/Users/manjujain./RestAssured/locator_expectedValue.properties (No such file or directory)">
      <![CDATA[java.io.FileNotFoundException: /Users/manjujain./RestAssured/locator_expectedValue.properties (No such file or directory)
at java.io.FileInputStream.open0(Native Method)
at java.io.FileInputStream.open(FileInputStream.java:195)
at java.io.FileInputStream.<init>(FileInputStream.java:138)
at BranchIO.com.Base.setupEn(Base.java:29)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:168)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- setupEn -->
  <testcase name="launchHomePage" time="0.000" classname="BranchIO.com.HomePageTest">
    <skipped/>
  </testcase> <!-- launchHomePage -->
  <testcase name="HomePage_team" time="0.000" classname="BranchIO.com.HomePageTest">
    <skipped/>
  </testcase> <!-- HomePage_team -->
  <testcase name="varifyEmployeeList" time="0.000" classname="BranchIO.com.HomePageTest">
    <skipped/>
  </testcase> <!-- varifyEmployeeList -->
  <testcase name="validateNumberOfEmployee" time="0.000" classname="BranchIO.com.HomePageTest">
    <skipped/>
  </testcase> <!-- validateNumberOfEmployee -->
  <testcase name="navigateHomePageWithGoogle" time="0.000" classname="BranchIO.com.HomePageTest">
    <skipped/>
  </testcase> <!-- navigateHomePageWithGoogle -->
  <testcase name="openNewtab" time="0.000" classname="BranchIO.com.HomePageTest">
    <skipped/>
  </testcase> <!-- openNewtab -->
</testsuite> <!-- BranchIO.com.HomePageTest -->
